docs/source/pythonapi/examples/mgxs-part-iv.ipynb

```diff
@@ -1272,27 +1272,27 @@
       "\n",
       " =======================>     TIMING STATISTICS     <=======================\n",
       "\n",
-      " Total time for initialization     =  4.3487E-02 seconds\n",
-      "   Reading cross sections          =  3.0800E-03 seconds\n",
-      " Total time in simulation          =  8.6325E+00 seconds\n",
-      "   Time in transport only          =  8.0901E+00 seconds\n",
-      "   Time in inactive batches        =  6.8907E-01 seconds\n",
-      "   Time in active batches          =  7.9435E+00 seconds\n",
-      "   Time synchronizing fission bank =  5.1527E-03 seconds\n",
-      "     Sampling source sites         =  3.5060E-03 seconds\n",
-      "     SEND/RECV source sites        =  1.5677E-03 seconds\n",
-      "   Time accumulating tallies       =  1.1610E-04 seconds\n",
-      " Total time for finalization       =  3.1340E-06 seconds\n",
-      " Total time elapsed                =  8.6927E+00 seconds\n",
-      " Calculation Rate (inactive)       =  72561.5 neutrons/second\n",
-      " Calculation Rate (active)         =  25177.9 neutrons/second\n",
+      " Total time for initialization     =  6.3715E-02 seconds\n",
+      "   Reading cross sections          =  2.8338E-02 seconds\n",
+      " Total time in simulation          =  7.4003E+00 seconds\n",
+      "   Time in transport only          =  7.3543E+00 seconds\n",
+      "   Time in inactive batches        =  6.4307E-01 seconds\n",
+      "   Time in active batches          =  6.7572E+00 seconds\n",
+      "   Time synchronizing fission bank =  4.9557E-03 seconds\n",
+      "     Sampling source sites         =  3.4097E-03 seconds\n",
+      "     SEND/RECV source sites        =  1.4729E-03 seconds\n",
+      "   Time accumulating tallies       =  1.0309E-04 seconds\n",
+      " Total time for finalization       =  3.1860E-06 seconds\n",
+      " Total time elapsed                =  7.4812E+00 seconds\n",
+      " Calculation Rate (inactive)       =  77751.8 neutrons/second\n",
+      " Calculation Rate (active)         =  29598.1 neutrons/second\n",
       "\n",
       " ============================>     RESULTS     <============================\n",
       "\n",
-      " k-effective (Collision)     =  1.02178 +/-  0.00213\n",
-      " k-effective (Track-length)  =  1.02205 +/-  0.00230\n",
-      " k-effective (Absorption)    =  1.02440 +/-  0.00198\n",
-      " Combined k-effective        =  1.02354 +/-  0.00179\n",
+      " k-effective (Collision)     =  1.02372 +/-  0.00236\n",
```

Comment:
I have no idea. I just re-ran and for some reason the values are now the same. (????)

- [x]